model/registry/process.yaml

```
@@ -11,6 +11,31 @@ groups:
         brief: >
           Process identifier (PID).
         examples: [1234]
+      - id: entity_id
+        type: string
+        stability: experimental
+        brief: >
+          A unique identifier of the process.
+        note: |
+          A globally unique identifier of the process. This can be any format
```

this is already covered in the brief
```suggestion
           This can be any format
```